.editorconfig

```diff
@@ -7,3 +7,12 @@ indent_style = space
 # TS files
 [*.ts]
 indent_size = 2
+
+# Fantomas (see https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md)
+[*.fs]
+indent_size=4
+max_line_length=140
+fsharp_space_before_uppercase_invocation=false
```

Comment:
Out of curiosity, why are you mentioning these settings here?
They have the same values as the default ones, so this won't do anything.

Reply:
@nojaf Yes, you are right. It seems I had a .editorconfig in a parent dir as defaults and forgot about it (scary). I'll remove it and simplify this file. Thanks for letting me know. I have wondered about that a couple of time.